mandatory/2-function-creation.js

```diff
@@ -50,25 +50,13 @@ function tidyUpStrings(arrayOfStrings) {
   const strArr = arrayOfStrings;
 // return strArr;
    
- const allItems = strArr.map((word) => word.toLowerCase()); //console.log(map2);
- return allItems;
-
-  //  const smlCap = strArr.map(word => word.toLowerCase());
-  //   console.log(smlCap);
-
-
-
-//  //Trim
-//   arrayOfStrings.map((strArr) => strArr.trim());
-//   console.log(arrayOfStrings());
+ const smlCap = strArr.map((word) => word.toLowerCase()); // .map() iterates through all items and turns into lowercase
+ const trimName = smlCap.map((word) => word.trim());     //  Iterates smlCap, & trims spaces with .trim() 
+ const noSlash = trimName.map((word) => word.replace('/', ''));  // Iterates through trimName & removes forward slash using replace .map() and .replace() methods
+ 
```

Comment:
Great use of map method 👍 very good explanations

Reply:
very complete explanation. great job Maha

Reply:
Thanks Omer and Maziar!  :)